app/decorators/reviewer/proposal_decorator.rb

```diff
@@ -25,4 +25,15 @@ def comment_count
   def internal_comments_style
     object.was_rated_by_person?(h.current_user) ? nil : "display: none;"
   end
+
+  def rating_tooltip
```

Comment:
So I noticed you have two methods that are identical.  This is duplication and could cause issues if someone only updates one of the methods and doesn't think to look for a second.  Can you think of a way to remove the duplication by putting it on one place?